### DISCUSSION TITLE
ftp/SSH access to Azuracast system folders

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#5042] [Q&A] [answered]
AzuraCast logs always report like this when AutoDj fetches a media file;
[ ("filename", "/var/azuracast/stations/radio_des_festivals/media/Country Franco/Jeannine_Savard_-_Tant_que_le_soleil_brillera_-_12_-_C'est_la_valse_de_mon_coeur.mp3") ]

Question is; why is it that I can't navigate to this folder or any other under /azuracast via Putty or Filezilla in root mode? In fact /var/azuracast comes up empty...

**Comment by ItsMitchh** [comment, marked as answer]
While the files are technically stored in the `/var/azuracast/stations` area, you cannot see them because the server is using the Docker Containers. To locate them, its simple, go to `/lib/var/docker/volumes/azuracast_station_data/` and you'll find your files or something like that. We could probably have a note but don't think its really worth it since it's within the docker container / volume.

**Reply by SpeedyCharly** [reply]
Thks Mitch!
